modules/compute/container-registry/test/dependencies.test.bicep

```diff
@@ -0,0 +1,113 @@
+param location string
+param name string
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2022-05-01' = {
+  name: 'dep${name}01'
```

Comment:
i'd prefix with something like this, for any resources that need to be globally unique

```bicep
${uniqueString(resourceGroup().id, deployment().name)}
```

Reply:
I've pushed a commit with some changes in the tests to ensure the a unique string is used, thanks!